vllm/v1/attention/backends/flash_attn.py

```diff
@@ -20,10 +20,13 @@
 logger = init_logger(__name__)
 
 
-class FlashAttentionBackend(AttentionBackend):
+class FlashAttentionBackend:
 
     accept_output_buffer: bool = True
 
+    def __init__(self, runner):
+        self.runner = runner
```

Comment:
you can probably update the `__init__` for AttentionBackend class such that all subclass can have this.